.github/workflows/ci-checks.yml

```diff
@@ -78,3 +78,31 @@ jobs:
             env:
                 TEST_TOKEN: ${{ secrets.TEST_TOKEN }}
             run: "python -m pytest --verbose tests/"
+    validate-documentation-examples:
+        runs-on: [ ubuntu-latest ]
+        needs: [black, pylint, flake8]
+        name: "Validate examples in documentation"
+        steps:
+        -   uses: actions/checkout@v2
+        -   name: "Install dependencies"
+            run: |
+                pip install notebook tqdm ipywidgets
```

Comment:
We are installing `tqdm` here and also in the `requirements_dev.txt`. One of them needs to be removed.